pyiron/lammps/control.py

```diff
@@ -789,6 +789,32 @@ def energy_pot_per_atom(self):
             self['dump___1'] += ' c_energy_pot_per_atom'
             self['dump_modify___1'] = self['dump_modify___1'][:-1] + ' %20.15g"'
 
+    def _set_group_by_id(self, group_name, ids):
+        if len(ids) < 1:
+            raise ValueError('Group ids must have at least length one, but got {}'.format(ids))
+        self['group___{}'.format(group_name)] = 'id {}'.format(' '.join(np.array(ids).astype(int).astype(str)))
+
+    def fix_move_linear_by_id(self, ids, velocity):
+        """
+        Displace atoms at each timestep. Creates a new group with a unique name based off the hash of the ids.
+
+        Args:
+            ids (list/numpy.ndarray): Integer ids of the atoms to move in the job's structure.
+            velocity (list/numpy.ndarray/tuple): The velocity in x-y-z-direction for the group. `None` arguments are OK.
+        """
+        if len(velocity) != 3:
+            raise ValueError('Velocity must have three components, but got {}'.format(velocity))
+        conversion = LAMMPS_UNIT_CONVERSIONS[self["units"]]["velocity"]
+        velocity = list(velocity)
+        for i, v in enumerate(velocity):
+            if v is None:
+                velocity[i] = 'NULL'
+            else:
+                velocity[i] = str(v * conversion)
```

Comment:
Don't we have to rotate the velocity? (MOUWAHHAHAHAHAHAAAA!)

Reply:
Nope, it's fine 😉 

It is, of course, not fine. But the rotation matrix gets passed in explicitly to the `LammpsControl.calc_*` methods from modules that know something about the rotation matrix. Here there is no interface outside the control module, so we're not obligated to provide rotation support -- if it goes wrong it's the user's fault 😈 
> Currently tucked away hidden in lammps.input.control with no direct job access as it's probably just for experts/not something you can (currently) do in other codes(?).